ui/component/or-asset-tree/src/or-add-asset-dialog.ts

```diff
@@ -104,27 +116,67 @@ export class OrAddAssetDialog extends LitElement {
             #mdc-dialog-form-add {
                 display: flex;
                 height: 600px;
-                width: 800px;
+                width: 1000px;
                 border-style: solid;
                 border-color: var(--or-app-color5, ${unsafeCSS(DefaultColor5)});
                 border-width: 1px 0;
             }
+
+            .msg {
+                display: flex;
+                justify-content: center;
+                align-items: center;
+                text-align: center;
+                height: 100%;
+                font-family: "Segoe UI", Helvetica,Arial,sans-serif,"Apple Color Emoji","Segoe UI Emoji","Segoe UI Symbol";
```

Comment:
Technically your message should be in a span, so the font family will come through by itself, but it's minor.